src/modules/jcms_article_fragment/jcms_article_fragment.install

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Contains jcms_article_fragment install/update functions.
+ */
+
+/**
+ * Implements hook_update_N().
+ */
+function jcms_article_fragment_update_8001() {
+  \Drupal::entityDefinitionUpdateManager()->applyUpdates();
```

Comment:
This should be handled with:

```
drush cim -y
```

which should be run as a deploy step in all environments. While it does no harm to add it, it should not be necessary so please remove.